scripts/docs/species_list.pl

```diff
@@ -360,14 +361,22 @@ sub round_count {
     $count_display = $count;
     $bg_class = $colours{'lot_million'}{'colour'};
   }
-  # From 100 million
-  elsif ($count =~ /^(\d{3}\d*)\d{6}$/) {
+  # From 100 million to 999.9 million
+  elsif ($count =~ /^(\d{3})\d{6}$/) {
     my $number = $1;
     $count = "$number M";
     $count_label = "Over $number million $type";
     $count_display = $count;
     $bg_class = $colours{'hundred_million'}{'colour'};
   }
+  # From 1 to 9.9 billion
+  elsif ($count =~ /^(\d)(\d)\d{8}$/) {
```

Comment:
Wouldn't it be better to future-proof this to also capture here 10 billion and above? Otherwise, if we forget to update this, those big numbers will be printed in full based on the `else` statement (which is worse).

```suggestion
  elsif ($count =~ /^(\d+)(\d)\d{8}$/) {
```

Reply:
Quite true, I have updated to include more than 10 billion in this regex.